--- rtracklayer/rtracklayer/DESCRIPTION~	2013-05-04 06:57:13.000000000 +0200
+++ rtracklayer/rtracklayer/DESCRIPTION	2013-05-10 14:37:24.622975892 +0200
@@ -5,7 +5,7 @@
 Depends: R (>= 2.10), methods, GenomicRanges (>= 1.11.12)
 Imports: XML (>= 1.98-0), BiocGenerics (>= 0.1.0), IRanges (>=
         1.15.37), GenomicRanges, Biostrings (>= 2.25.6), BSgenome (>=
-        1.23.1), zlibbioc, RCurl (>= 1.4-2), Rsamtools (>= 1.7.3)
+        1.23.1), RCurl (>= 1.4-2), Rsamtools (>= 1.7.3)
 Suggests: humanStemCell, microRNA (>= 1.1.1), genefilter, limma,
         org.Hs.eg.db, BSgenome.Hsapiens.UCSC.hg19,
         TxDb.Hsapiens.UCSC.hg19.knownGene, hgu133plus2.db
--- rtracklayer/rtracklayer/NAMESPACE~	2013-05-04 02:02:40.000000000 +0200
+++ rtracklayer/rtracklayer/NAMESPACE	2013-05-10 14:37:38.086445477 +0200
@@ -1,7 +1,5 @@
 useDynLib(rtracklayer, .registration = TRUE)
 
-import(zlibbioc)
-
 importFrom("stats", offset)
 
 importFrom("XML", getNodeSet, xmlValue, xmlAttrs, htmlTreeParse,
